src/main/resources/i18n/Enums_en.properties

```diff
@@ -19,6 +19,18 @@ accelerator.DeliverableCategory.SupplementalFiles=Supplemental Files
 accelerator.DeliverableCategory.VerraNonPermanenceRiskToolNPRT=Verra Non-Permanence Risk Tool (NPRT)
 # Noun, as in a legal document
 accelerator.DeliverableType.Document=Document
+accelerator.ScoreCategory.Carbon=Carbon
+accelerator.ScoreCategory.Finance = 'Finance'
```

Comment:
This should just be `=Finance` rather than ` = 'Finance'` so it doesn't show up as a diff when Phrase uploads translations of this file. It treats the English files as translations too and it generates new copies of them and includes them in the PRs if there are changes.

Reply:
My mistake! I overlook the formatting of those.